README.md

```diff
@@ -515,8 +515,9 @@ command line option. The option takes an argument of the form `pattern:syntax` w
 the absolute file path. The `syntax` part is the full name of a supported language
 (use `bat --list-languages` for an overview).
 
-Note: You probably want to use this option as an entry in `bat`s configuration file instead
-of passing it on the command line (see below).
+*Note:* You probably want to use this option as [an entry in `bat`'s configuration file](#configuration-file)
+for persistence, instead of passing it on the command line as a one-off, as generally
+you'd just use `-l` if you want to manually specify a language for a file.
```

Comment:
```suggestion
**Note:** You probably want to use this option as [an entry in `bat`'s configuration file](#configuration-file)
for persistence instead of passing it on the command line as a one-off. Generally
you'd just use `-l` if you want to manually specify a language for a file.
```

I'd link "use `-l`" if there were a ReadMe anchor for it. The closest I can get is How to Use. Also, there are other places were "Note:" is bolded, so I'm suggesting this to be consistent.